docs/app/css/style.css

```diff
@@ -51,6 +51,31 @@ ul li:first-child {
   margin-top: 0;
 }
 
+ul.skip-links li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+ul.skip-links li a {
+  background-color: #fff;
+  display: block;
+  font-size: 1.25em;
+  margin: 0.5em;
+  opacity: 0;
```

Comment:
So by using opacity:0, it is not visible... but still rendered. 
Would `display:none;` be a better choice?